src/amity.py

```diff
@@ -0,0 +1,498 @@
+import os
+import re
+import click
+from random import randint
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+
+from .room import Office, LivingSpace
+from .person import Fellow, Staff
+from .db_organisation.db_logic import DbManager, Base, Rooms, Persons
+
+
+
+class Amity(object):
+    """
+    Amity class implementing the logic of a facility where it has rooms which can be offices or living spaces. An office can occupy a maximum of 6 people. A living space can inhabit a maximum of 4 people
+    A person to be allocated could be a fellow or staff. Staff cannot be allocated living spaces. Fellows have a choice to choose a living space or not.
+
+    """
+
+    offices = {}
+    living_spaces = {}
+    fellows = {}
+    staff = {}
+
+    def create_room(self, name, typ):
+        """
+        Creates a room in Amity
+
+        """
+        if typ == "O":
```

Comment:
Use descriptive variable names to enable someone understand easily what the variable holds